frontend/src/components/StaticNodeDetails.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2018-2019 Google LLC
```

Comment:
One more overall comment: Is it possible to implement something that would work even if a pipeline step is used multiple times? In this case, one execution should not override the data written by another. 

For example, Ning is planning to add loop support: https://docs.google.com/document/d/12KHoEGe3o-i2WyzaU2JPXp3GQL3_BavGuh-KYBMUdQ8/edit?ts=5c5b2627#heading=h.bhvs46afvzxg

Reply:
@vicaire 
> One more overall comment: Is it possible to implement something that would work even if a pipeline step is used multiple times? In this case, one execution should not override the data written by another.
> 
> For example, Ning is planning to add loop support: https://docs.google.com/document/d/12KHoEGe3o-i2WyzaU2JPXp3GQL3_BavGuh-KYBMUdQ8/edit?ts=5c5b2627#heading=h.bhvs46afvzxg

I will amend the design doc in #801 so we use `PipelineParam` instances in the K8s resource specs referred to by `PipelineVolume` and `PipelineVolumeSnapshot` instances. This should make them usable in loops.

We don't have access to an implementation of loops yet, so we cannot verify our code, but we are confident this will work, based on our use of `PipelineParam` instances. Let's talk more about it in #801, if you feel there is something that can cause problems with your implementation.